collections/_infographics/policies/emission-pledges.md

```diff
@@ -12,4 +12,34 @@ data-orig:
   - [ "Our Jupyter notebook", "https://github.com/faktaoklimatu/data-analysis/blob/master/notebooks/net-zero-pledges.ipynb" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/emisni-zavazky" %}
+## How to read an infographic?
+{% comment %}
+{% include preview-box.html
+    title="What is carbon neutrality?"
+    text="We explain exactly what the zero carbon commitment means and how the term can be confusing."
+    slug="uhlikova-neutralita"
+%}
+{% endcomment %}
+This infographic shows the share of carbon-neutral countries in global CO<sub>2</sub> emissions in 2018. Excluding the Land Use, Land Use Change and Forestry sector (*LULUCF), all CO<sub>2</sub> emissions, including international maritime and aviation transport. **As of March 2022, 87% of global emissions come from countries moving towards carbon neutrality**, with 131 countries total.
```

Comment:
small: no need for a * in (*LULUCF); not common in English acronyms in my experience.

Reply:
the 131 countries total is ambiguous here. Rephrase as:

**As of March 2022, the 131 countries that are moving towards carbon neutrality are responsible for 87% of global emissions**.

Reply:
I did not intend to keep the asterisk. Removed it. Thanks for catching this.

Reply:
I updated text in the paragraph as suggested.